src/main/scala/io/iohk/ethereum/jsonrpc/JsonRpcError.scala

```diff
@@ -5,10 +5,12 @@ import org.json4s.JsonAST.JValue
 case class JsonRpcError(code: Int, message: String, data: Option[JValue])
 
 // scalastyle:off magic.number
+// scalastyle:off public.methods.have.type
 object JsonRpcErrors {
   val ParseError = JsonRpcError(-32700, "An error occurred on the server while parsing the JSON text", None)
   val InvalidRequest = JsonRpcError(-32600, "The JSON sent is not a valid Request object", None)
   val MethodNotFound = JsonRpcError(-32601, "The method does not exist / is not available", None)
-  val InvalidParams = JsonRpcError(-32602, "Invalid method parameters", None)
+  def InvalidParams(msg: String = "Invalid method parameters") = JsonRpcError(-32602, msg, None)
```

Comment:
Normally, yes. But here I preferred the capitalised version for consistency with the vals. I don't have strong feeling one way or the other, would you like me to change this?